lib/js/components/font-loading.mjs

```diff
@@ -131,16 +131,18 @@ export class FontSelect extends _BaseComponent {
           , optGroups = {}
           , createOptGroup = name=>{
                 const optgroup = this._domTool.createElement('optgroup');
+                const legend = this._domTool.createElement('legend');
```

Comment:
I'm using Firefox on Linux, an the `<legend>` elements in the `<optgroup>s` don't render at all. A fallback that works for me in this case is to also define the `label=` attribute within the `<optgroup>` . In Chrome the `label=` is ignored in favor of the `<legend>`, in Firefox the `label=` is used, the `<legend>` is ignored anyway. Also tried on Edge on Linux.

This would require testing on a Mac/Safari at least as well.